keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -172,9 +184,24 @@ available memory are more probable to require restarts.
 If more than one resource type with different policies are updated, then
 RestartContainer policy takes precedence over NoRestart policy.
 
-Additionally, if RestartPolicy is 'Never', ResizePolicy should be set to
+Additionally, if RestartPolicy is 'Never', ResizePolicy must be set to
 NoRestart in order to pass validation.
 
+<<[UNRESOLVED]>>
+Should something like RuntimeClass be allowed to say "no in-place restarts are
+allowed", so that users would get a synchronous error if they set this?  Or is
+that overkill?`
```

Comment:
The impression I got from sig-node discussions is that runtimes are expected to honor the CRI APIs. Granted that even a minor CRI extension proposed in this PRs [sister KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20191025-kubelet-container-resources-cri-api-changes.md) won't happen overnight, but we would expect that runtimes support over the span of one or two release after this goes beta or GA. This would be nice to have in the interim though. Let me bring it up in the next sig-node meeting and see what people think.

Reply:
For clarity: I am not sure it's worthwhile, so a valid answer will be "no, because..."